.github/workflows/sandbox-creation.yml

```
@@ -0,0 +1,127 @@
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  CheckTokenInSecretsManager:
+    runs-on: ubuntu-latest
+    outputs:
+      rotate_test_needed: ${{ steps.determine_test_rotation.outputs.rotate_test_needed }}
+      rotate_prod_needed: ${{ steps.determine_prod_rotation.outputs.rotate_prod_needed }}
```

actionlint (1.7.4)

property "determine_prod_rotation" is not defined in object type {determine_test_rotation: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)